tools/fluent/basic_syntax.md

```diff
@@ -113,19 +106,30 @@ emails = { $unreadEmails ->
 
 While in most cases messages will have a single value, it’s also possible to define multiple **variants**: different values, each one associated to a key, that can be selected when referencing the message ID from another message. Unlike selectors, the message doesn’t depend on the value of an external variable or function.
 
-This can be useful, for example, to decline names according to grammatical case.
+This can be useful, for example, to translate according to grammatical case or gender. The following example combines a private message (`-creature-elf`) and variants.
+
+In English:
+
+```PROPERTIES
+-creature-elf = an Elf
+
+you-found-elf = You found { -creature-elf }.
+elf-found-you = Oops, { -creature-elf } found you.
```

Comment:
I know it's hard to find good examples which are not complex. Sadly, I don't think this example is a good one. As soon as you hard-code `elf` into the message identifier `you-found-elf`, the good practice is to just spell out `an elf` in the same message rather than factor it out. So: 

```properties
you-found-elf = You found an elf.
```

Reply:
Maybe we should drop the section on variants and bring it back when the brand name localization requirements are relaxed?

Reply:
I know it's a bad example in terms of localizability, but the target here is localizers (to understand the syntax), not developers. Sure, I'd be happy to find a better example :-\